### ISSUE TITLE
Plugin not logging in anymore

### ISSUE THREAD

**Issue by jcouzy** [homebridge-eufy-security/plugin#320]
### What happened?

A bug happened! I receive an error not connected in the plugin logs.
Everytime I login  need to enter a Captcha code.

### Device Type

Doorbell (Supported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v12 (Supported)

### Operating System

Raspbian 10+ (Supported)

### Relevant log output

```shell
Not connected can't continue!
```

### config

```shell
{
            "platform": "EufySecurity",
            "ignoreStations": [],
            "ignoreDevices": [],
            "cleanCache": true,
            "username": "xx@xx.com",
            "password": "xxxxx",
            "country": "NL",
            "deviceName": "",
            "cameras": [
                {
                    "serialNumber": "T8210P6422411FF8",
                    "rtsp": false,
                    "enableCamera": true,
                    "videoConfig": {
                        "audio": true,
                        "audioSampleRate": 0
                    },
                    "talkback": true,
                    "snapshotHandlingMethod": 2,
                    "refreshSnapshotIntervalMinutes": 5,
                    "immediateRingNotificationWithoutSnapshot": true
                },
                {
                    "serialNumber": "T8410P42224801EB",
                    "enableCamera": "true",
                    "useCachedLocalLivestream": true,
                    "videoConfig": {
                        "audio": true,
                        "audioSampleRate": 0
                    },
                    "talkback": true,
                    "snapshotHandlingMethod": 2,
                    "refreshSnapshotIntervalMinutes": 5
                }
            ],
            "_bridge": {
                "username": "xxxx",
                "port": 39401
            },
            "pollingIntervalMinutes": 1,
            "CameraMaxLivestreamDuration": 120
        }
```

**Comment by opala41** [comment]
Same here, unfortunately. I tried to remove the 2FA option, created a secondary account and even contacted Eufy. They informed the captcha is normal… 

I noticed that a list of random Android models assigned as “trusted” devices populate in the Eufy app everytime I try to log into using the plugin. It’s always a different model, I don’t know if this is the culprit, triggering their system about possible invasion attempts. 

Would you take a look please? Thanks! Your great plug in is one of the reasons for me to use my 15+ Eufy devices.

**Comment by wyse2000** [comment]
i have de same problems. 2Fa authentication

**Comment by pillemats** [comment]
same here

**Comment by pillemats** [comment]
is there any solution? The problem is weeks ago and the plugin is not useable

**Comment by ibmaster** [comment]
Works for me, just have to keep doing the captcha login process and Homebridge restart until it passes

**Comment by pillemats** [comment]
i dont get a captach login

**Comment by pillemats** [comment]
This is what happens when i try to login.... (nothing more.....)

<img width="1281" alt="Bildschirmfoto 2023-06-24 um 08 15 04" src="https://github.com/homebridge-eufy-security/plugin/assets/26164977/5b85c92d-f7a1-46ea-96bc-cfc763f6bd55">
<img width="1050" alt="Bildschirmfoto 2023-06-24 um 08 15 27" src="https://github.com/homebridge-eufy-security/plugin/assets/26164977/55bc0594-611f-4580-b1e5-dc1fb899caba">

**Comment by sjuut** [comment]
+1, I am currently facing the same issue.

**Comment by ckangnz** [comment]
+1 on this.... i was having issue not seeing my eufy 3 cams and logging in and out, and got stuck with this issue

**Comment by pillemats** [comment]
This issue is weeks ago now

**Comment by funtush** [comment]
Same error. Can't connect Looks like Captcha is required everytime you restart plugin.

**Comment by yennik** [comment]
I resolved this issue by logging into the dedicated Eufy account in the app and accepting the invite manually; I was relying on the plugin to auto-accept the invite as is documented, but doesn't seem to be the case.

First time setup for me and I was encountering this issue where I was getting continuous CAPTCHAs. I tried the beta v2.2.0, uninstalling, rebooting, uninstalling and completely removing the directory. No success until I logged into the Homebridge dedicated account I created and manually accepted the home invite from my primary Eufy account.
